rosbag2_storage/include/rosbag2_storage/storage_options.hpp

```diff
@@ -60,15 +66,18 @@ struct StorageOptions
   std::unordered_map<std::string, std::string> custom_data{};
 };
 
+void declare_storage_options_r_params(std::shared_ptr<rclcpp::Node> nh, StorageOptions & so);
+void declare_storage_options_rw_params(std::shared_ptr<rclcpp::Node> nh, StorageOptions & so);
+
 }  // namespace rosbag2_storage
 
 namespace YAML
 {
 template<>
 struct ROSBAG2_STORAGE_PUBLIC convert<rosbag2_storage::StorageOptions>
 {
-  static Node encode(const rosbag2_storage::StorageOptions & storage_options);
-  static bool decode(const Node & node, rosbag2_storage::StorageOptions & storage_options);
+  static Node encode(const rosbag2_storage::StorageOptions & so);
+  static bool decode(const Node & node, rosbag2_storage::StorageOptions & so);
```

Comment:
Please revert these changes 
```suggestion
  static Node encode(const rosbag2_storage::StorageOptions & storage_options);
  static bool decode(const Node & node, rosbag2_storage::StorageOptions & storage_options);
```